File: 은지/LV1/같은 숫자는 싫어.py
def solution(arr):
    answer = []
    # 과거 숫자 초기화
    pre = -1
    for x in arr:
        # 과거 숫자와 현재 숫자 다른 경우
        if pre != x:
            answer.append(x)
            # 과거 숫자 변경
            pre = x
    return answer

File: 태성/LV1/0616/같은숫자는싫어.js
const arr = [1,1,3,3,0,1,1]	

function solution(arr){
    var answer = [];

    var pre = arr[0]
    for (i=1; i<arr.length; i++) {
      if (pre === arr[i]) {
        pre = arr[i]
      } else {
        answer.push(pre)
        pre = arr[i]
      }

    }

    answer.push(pre)
    
    return answer;
}

console.log(solution(arr))